development-docs/RELEASE.md

```diff
@@ -59,7 +59,7 @@ The release process should normally look like this:
         ```
         For example, for Strimzi 0.45.0 release, if you unpacked the release artifacts to `./strimzi-0.45.0-rc1/` and have the Strimzi website checkout in `strimzi.github.io/`, you would run:
         ```
-        helm repo index ./strimzi-0.45.0/ --merge ./strimzi.github.io/charts/index.yaml --url https://github.com/strimzi/strimzi-kafka-operator/releases/download/0.45.0/
+        helm repo index ./strimzi-0.45.0-rc1/ --merge ./strimzi.github.io/charts/index.yaml --url https://github.com/strimzi/strimzi-kafka-operator/releases/download/0.45.0/
```

Comment:
This seems wrong:
* The versions should be the same in the command
* You do this only for the final release, not for RCs. So why add `-rc1`?

Reply:
Sorry I meant to do make the change the other way around, the text line above the command line was wrong containing `-rc1` because you are right this command is for GA only. I fixed it now.